<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Airport</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>431</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>SAVE INFO</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QSpinBox" name="spinBox_2"/>
     </item>
     <item row="2" column="0">
      <widget class="QSpinBox" name="spinBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>WORK</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
